docs/release-notes.rst

```diff
@@ -3,6 +3,47 @@
 Release Notes
 #############
 
+v0.3.2
```

Comment:
Usually we do this in a separate PR before release, so let's skip this for now

Reply:
ok -- i'll move the commits with the release notes into a separate PR